lib/plugins/aws/lib/get-shared-direct-stack-action-params.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+module.exports = {
+  getSharedDirectStackActionParams({ templateUrl, templateBody }) {
```

Comment:
Shouldn't we also use it in `getCreateChangeSetParams` ?

Reply:
I was thinking about it as it shares some similar parameters, but I've intended this tool to be only used for direct stack operations. Do you think we should make it more generic to also be reused for the `getCreateChangeSetParams`?

Reply:
If I see correctly all of this function's logic is reflected in `getCreateChangeSetParams` (which just adds `ChangeSetName` extra).

In light of that, I would generalize name fo this util  to `getSharedStackActionParams` and reuse it in `getCreateChangeSetParam` (we may also drop `getCreateChangeSetParam` entirely, as keeping it for just one property setting seems overkill)

Reply:
Or do you see that both approaches may diverge in a future (that would be a valid reason to keep it separate)

Reply:
I wanted to intentionally keep these both approaches separated as much as possible if we ever decide to get rid of one of them e.g. for maintenance purposes, but I guess the crossover here would be really small, so I guess we can have a more generic shared util here. I'll update it